substrate/frame/support/src/traits/try_runtime/mod.rs

```diff
@@ -95,6 +96,8 @@ impl sp_std::str::FromStr for Select {
 }
 
 /// Select which checks should be run when trying a runtime upgrade upgrade.
+#[bitflags]
+#[repr(u64)]
 #[derive(codec::Encode, codec::Decode, Clone, Debug, Copy, scale_info::TypeInfo)]
 pub enum UpgradeCheckSelect {
```

Comment:
`None` and `All` doesn't make sense anymore with `bitflags`.

Reply:
`None` and `All` are still there? You can just create a `all` and `none` function on the enum.  
These bitflags are supposed to make the code easier, not weirder.

Reply:
For sure they are there. However a `none` with a bitflag is just `0` and `all` is just all possible bits set. I was also speaking about the variants and not the functions the macro generates, because the functions will do what I said before.

Reply:
Yes exactly. I was just commenting on the fact that the last commit of the MR creator did not address this 🙃